convert2rhel/actions/report.py

```diff
@@ -264,6 +269,24 @@ def format_report_section_heading(status_code):
     return heading
 
 
+def find_highest_report_level(results):
+    """
+    Gather status codes from messages and result. We are not seeking for
+    differences between them as we want all the results, no matter from where
+    they come.
+    """
```

Comment:
Please, if you can, update the docstring to include the param/type/return/rtype. 

```suggestion
    they come.
    
    :param results: ...
    :type results: ...
    :return: ...
    :rtype: ...
    """
```